### ISSUE TITLE
A11y_GradeB_VSCode_Keyboard shortcut reads words together - Blind: Arrow key navigation to row Find the instructions to use " space or enter" to change the shortcut key is not read

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#182491]
**GitHubTags**:#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG3.3.2;#DesktopApp;#BM-VisualStudioCode-Win32-Nov2021;#NVDA;#Win32;#Win11;#AINotInScope;

## Environment Details:​
Application Name: Visual Studio Code
Version: 1.79.0-insider (user setup)
OS build 25300.1000
JAWS 2023.2303.144

## Repro Steps:

1. Launch Visual Studio Code insiders
2. Navigate to settings and press enter
3. tab to keyboard shortcut and press enter
4. use down key and navigate to a row
5. start JAWS

## Actual:
Arrow key navigation to row Find the instructions to use " space or enter" to change the shortcut key is not read

## Expected:
Arrow key navigation to row Find the instructions to use " space or enter" to change the shortcut key should be read

## User Impact:
screen reader users will not know how to change the keys

## Attachment
[Hint for screen reader would help-jaws.webm](https://github.com/microsoft/vscode/assets/24580570/4ac9dd8b-03a8-49c9-9ee4-498569cb5f87)

**Comment by sandy081** [comment]
@meganrogge Would you like to look into this and give insights or what is expected to be done?

**Comment by meganrogge** [comment]
I think this is a great use case for an `accessibility.verbosity` setting and `aria-label` hint like `use space or enter to change the keybinding`
